bootstrap.sh

```diff
@@ -12,32 +12,32 @@ function get_host_os {
     OSName=$(uname -s)
     case $OSName in
         Linux)
-            __HostOS=Linux
+            __HostOS=linux
             ;;
 
         Darwin)
-            __HostOS=OSX
+            __HostOS=osx
             ;;
 
         FreeBSD)
-            __HostOS=FreeBSD
+            __HostOS=freebsd
             ;;
 
         OpenBSD)
-            __HostOS=OpenBSD
+            __HostOS=openbsd
             ;;
 
         NetBSD)
-            __HostOS=NetBSD
+            __HostOS=netbsd
             ;;
 
         SunOS)
-            __HostOS=SunOS
+            __HostOS=sunos
```

Comment:
Haven't seen this RID before.  Is it going to be a problem that we don't have a mapping for this in runtime.json?